content/doc/guides/using-common-filters.dmark

```diff
@@ -23,6 +23,76 @@ title: "Using common filters"
 
   #p For more information, see the %ref[item=/doc/reference/filters.*,frag=sass]{%code{:sass} filter reference}.
 
+  #section %h{Support Sass and SCSS}
+    #p Sass has two syntaxes: Sass and SCSS. To support both, you can use curly braces in a compilation rule’s pattern to match both extensions:
+
+    #listing[lang=ruby]
+      compile '/css/*.{sass,scss%}' do
+        syntax = @item.identifier.ext.to_sym
+        filter :sass, syntax: syntax, style: :compact
+      end
+
+    #p The %code{:sass} filter is passed the %code{:syntax} option, which in the example above is derived from the item’s identifier.
+
+  #section %h{Using partials}
+    #p Partials are Sass files that don’t have an output file, but only serve to be included in another Sass file. By convention, filenames of partials start with an underscore (e.g. %filename{css/_layout.scss}), which makes it possible for them to be handled specially within Nanoc:
+
+    #listing[lang=ruby]
+      ignore '/css/_*.sass'
+
+      compile '/css/*.sass' do
+        filter :sass, syntax: :sass, style: :compact
+      end
+
+    #p With the rules defined above, you will be able to import Sass partials:
+
+    #listing[lang=sass]
+      @import "/css/**/_layout.*"
+
+  #section[id=using-sass-source-maps] %h{Using Sass source maps}
+    #p Nanoc’s Sass filter supports %ref[url=http://thesassway.com/intermediate/using-source-maps-with-sass]{source maps}, which makes debugging Sass easier. To generate a source map for Sass, use the %code{:sass_sourcemap} filter:
+
+    #listing[lang=ruby]
+      [:default, :sourcemap].each do |rep_name|
+        compile '/css/**/*.sass', rep: rep_name do
+          path = @item.identifier.without_ext + '.css'
+          options = { syntax: :sass, style: :compact, sourcemap_path: path + '.map' %}
+
+          case rep_name
+          when :default
+            filter :sass, options
+            write path
+          when :sourcemap
+            filter :sass_sourcemap, options
+            write path + '.map'
+          end
+        end
+      end
+
+    #p When using the %code{:sass_sourcemap} filter, be sure to invoke it with the same options as the %code{:sass} filter.
+
+  #section %h{Using %code{nanoc()} in Sass}
+    #p Nanoc’s Sass filter defines a %code{nanoc()} Sass function, which allows you to evaluate Ruby code from Sass within the compilation context. It takes a string containing Ruby code.
+
+    #p For example, assume a configuration file with the following contents:
+
+    #listing[lang=yaml]
+      title: Hello world!
```

Comment:
Maybe
```
    #listing[lang=yaml]
      # nanoc.yaml
      title: Hello world!
```
like on the reference page?

Reply:
Hmm, I think in this case it’s redundant, because the paragraph above mentions explicitly that it’s the configuration file. I do think I want to make the paragraph itself a little more clear, though.